### ISSUE TITLE
If I type a hashtag into a comment should it link to the hashtag? 

### ISSUE THREAD

**Issue by karchbold** [jjcm/soci#88]
I expected it would 

![image](https://user-images.githubusercontent.com/49083233/116836574-60495080-ac0a-11eb-836d-b322a8b4a5bf.png)

**Comment by jjcm** [comment]
Good call. Will have to implement this and upgrade the version of quill: https://github.com/weavy/quill-autoformat
